model/database/metrics.yaml

```diff
@@ -133,3 +134,39 @@ groups:
     attributes:
       - ref: db.client.connection.pool.name
```

Comment:
what does client connection pool name mean for Cosmos DB?

Reply:
this attribute is not related to CosmosDB. Hence, this attribute won't be there